node/src/client.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020-2022 Manta Network.
+// This file is part of Manta.
+//
+// Manta is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Manta is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Manta.  If not, see <http://www.gnu.org/licenses/>.
+
+//! RuntimeApi for client
+
+use manta_primitives::types::{AccountId, Balance, Block, Index as Nonce};
+use session_key_primitives::{AuraId, NimbusId};
+use sp_application_crypto::AppKey;
+use sp_core::Pair;
+use sp_runtime::traits::BlakeTwo256;
+
+/// RuntimeApiCommon + RuntimeApiNimbus: nimbus
+/// RuntimeApiCommon + RuntimeApiAura: aura
+///
+/// Common RuntimeApi trait bound
+pub trait RuntimeApiCommon:
+    sp_api::Metadata<Block>
+    + sp_api::ApiExt<Block>
+    + sp_block_builder::BlockBuilder<Block>
+    + sp_offchain::OffchainWorkerApi<Block>
+    + sp_session::SessionKeys<Block>
+    + sp_transaction_pool::runtime_api::TaggedTransactionQueue<Block>
+where
+    <Self as sp_api::ApiExt<Block>>::StateBackend: sp_api::StateBackend<BlakeTwo256>,
+{
+}
+
+/// Extend RuntimeApi trait bound for Nimbus
+pub trait RuntimeApiNimbus:
+    pallet_transaction_payment_rpc::TransactionPaymentRuntimeApi<Block, Balance>
+    + frame_rpc_system::AccountNonceApi<Block, AccountId, Nonce>
+    + cumulus_primitives_core::CollectCollationInfo<Block>
+    + sp_consensus_aura::AuraApi<Block, AuraId>
+    + nimbus_primitives::AuthorFilterAPI<Block, NimbusId>
+    + nimbus_primitives::NimbusApi<Block>
+{
+}
+
+/// Extend RuntimeApi trait bound for Aura
+pub trait RuntimeApiAura<AuraId: AppKey>:
```

Comment:
This trait isn't used anywhere in this PR?

Reply:
remove for now, previous I was trying change service_aura.rs, but it seems will be replace with nimubus, so yeah we don't need `RuntimeApiAura`.